lib/Runtime/Library/JavascriptPromise.cpp

```diff
@@ -267,6 +268,204 @@ namespace Js
+            remainingElementsWrapper->remainingElements--;
+            // 7.d.iii. If remainingElementsCount.[[Value]] is 0, then
+            if (remainingElementsWrapper->remainingElements == 0)
+            {
+                // 7.d.iii.1 Let error be a newly created AggregateError object.
+                JavascriptError* pError = library->CreateAggregateError();
+                JavascriptError::SetErrorsList(pError, errors, scriptContext);
+                JavascriptError::SetErrorMessage(pError, JSERR_PromiseAllRejected, _u(""), scriptContext);
+                JavascriptExceptionOperators::Throw(pError, scriptContext);
+            }
+        }
+        catch (const JavascriptException& err)
+        {
+            // 8. If result is an abrupt completion, then
+            //   a. If iteratorRecord.[[Done]] is false, set result to IteratorClose(iteratorRecord, result).
+            //   b. IfAbruptRejectPromise(result, promiseCapability).
+            JavascriptExceptionObject* exception = err.GetAndClear();
+            TryRejectWithExceptionObject(exception, promiseCapability->GetReject(), scriptContext);
+        }
+
+        return promiseCapability->GetPromise();
+    }
+
+    Var JavascriptPromise::EntryAnyRejectElementFunction(RecyclableObject* function, CallInfo callInfo, ...)
+    {
+        ScriptContext* scriptContext = function->GetScriptContext();
+        PROBE_STACK(scriptContext, Js::Constants::MinStackDefault);
+        ARGUMENTS(args, callInfo);
+        Assert(!(callInfo.Flags & CallFlags_New));
+
+        JavascriptLibrary* library = scriptContext->GetLibrary();
+        Var undefinedVar = library ->GetUndefined();
+        Var x = args.Info.Count > 1 ? args[1] : undefinedVar;
+
+
+        // 1. Let F be the active function object.
+        JavascriptPromiseAnyRejectElementFunction* anyRejectElementFunction = VarTo<JavascriptPromiseAnyRejectElementFunction>(function);
+
+        // 2. Let alreadyCalled be F. [[AlreadyCalled]].
+        // 3. If alreadyCalled. [[Value]] is true, return undefined.
+        if (anyRejectElementFunction->IsAlreadyCalled())
+        {
+            return undefinedVar;
+        }
+
+        // 4. Set alreadyCalled.[[Value]] to true.
+        anyRejectElementFunction->SetAlreadyCalled(true);
+
+        // 5. Let index be F.[[Index]].
+        uint32 index = anyRejectElementFunction->GetIndex();
+
+        // 6. Let errors be F.[[Errors]].
+        JavascriptArray* errors = anyRejectElementFunction->GetValues();
+
+
+        // 7. Let promiseCapability be F.[[Capability]].
+        JavascriptPromiseCapability* promiseCapability = anyRejectElementFunction->GetCapabilities();
+
+        // 9. Set errors[index] to x.
+        errors->DirectSetItemAt(index, x);
+
+        // 8. Let remainingElementsCount be F.[[RemainingElements]].
+        // 10. Set remainingElementsCount.[[Value]] to remainingElementsCount.[[Value]] - 1.
```

Comment:
Are you doing this step?

The following case currently doesn't print anything and this step being missing is my guess for why:
```js
Promise.any([Promise.reject(5)]).catch(e => print(e))
```

Reply:
Yep. But it's caused by another problem... :Xp